lib/class/update.class.php

```diff
@@ -4023,9 +4023,14 @@ public static function update_380013()
     {
         $retval = true;
 
+        // disable video
         $sql = "UPDATE `preference` SET `value`=0 WHERE `preference`.`name`='allow_video'";
```

Comment:
@lachlan-00, I think this should only be done on a fresh installation, as it would confuse users if the video function is suddenly not working anymore.
If my memory is not completely broken; there is a way to achieve this.